endpoints/secscan.py

```diff
@@ -11,3 +12,12 @@
 @anon_allowed
 def internal_ping():
     return make_response("true", 200)
+
+
+@secscan.route("/_backfill_status")
+@anon_allowed
+def manifest_security_backfill_status():
+    manifest_count = Manifest.select().count()
+    mss_count = ManifestSecurityStatus.select().count()
+    return jsonify({"backfill_percent": mss_count / float(manifest_count),})
```

Comment:
I mean this is really just reporting what percent of manifests in the registry have been scanned. Could be used not only for tracking backfill, but in general as well.

Reply:
That's the point, though: we'll ideally remove this once done